okhttp/src/main/java/okhttp3/RealCall.java

```diff
@@ -65,6 +66,16 @@ protected RealCall(OkHttpClient client, Request originalRequest) {
     }
   }
 
+  private void captureCallStackTrace() {
+    StackTraceElement[] stackTrace = Thread.currentThread().getStackTrace();
```

Comment:
Hmmm...

Android’s libcore has CloseGuard:
https://android.googlesource.com/platform/libcore/+/master/dalvik/src/main/java/dalvik/system/CloseGuard.java

Here’s how it’s used:
https://android.googlesource.com/platform/libcore/+/android-4.3_r1/luni/src/main/java/java/io/FileInputStream.java

Is there a reasonable way to detect if we’re in a development build? Maybe we can add this to Platform?
https://stuff.mit.edu/afs/sipb/project/android/docs/reference/android/os/Debug.html#isDebuggerConnected()